### PR TITLE
Updated exception recipient for master on production

### PR DESCRIPTION
**Pull request by holmesie**

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -81,7 +81,7 @@
   config.middleware.use ExceptionNotification::Rack, email: {
     email_prefix: "[Bonnie] ",
     sender_address: %{"Bonnie (#{APP_CONFIG['hostname']})" <bonnie@#{APP_CONFIG['hostname']}>},
-    exception_recipients: %w{bonnie-feedback-list@lists.mitre.org},
+    exception_recipients: %w{bonnie-production-errors-list@LISTS.MITRE.ORG},
```
Comment (mayerm94) [inline config/environments/production.rb:84]: any reason why LISTS.MITRE.ORG is capitalized?

Comment (c-monkey) [reply config/environments/production.rb:84]: I don't think it is case sensitive. I don't understand why bonnie-feedback-list is sent to the tacoma list, but my real question is: how do we test this?

Comment (c-monkey) [reply config/environments/production.rb:84]: We decided @holmesie is going to load production onto the Bonnie Alpha Staging machine (...245) so we can test this email.

Comment (holmesie) [reply config/environments/production.rb:84]: Pull request example emails sent and received 1:44pm EDT, check your inboxes for verification

**Review by c-monkey — APPROVED** [review]
emails look good

**Pull request merged**